server/extension/extension.go

```diff
@@ -640,7 +675,13 @@ func (m *Manager) CallExtension(extName string, registry ProxyRegistry) func(htt
 			return
 		}
 
-		proxy, err := findProxy(registry, extName, app.Spec.Destination)
+		proxyRegistry, ok := m.ProxyRegistry(extName)
+		if !ok {
+			m.log.Warnf("proxy extension warning: attempt to call unregistered extension: %s", extName)
```

Comment:
Maybe don't echo back the `extName`? Could be an XSS vector if the plugin author unwisely chooses to dump the raw response text into the page.

Reply:
I already sanitized the `extName` following the XSS linter suggestion. If you feel strong about removing this from the logs I can do it but I think it is a valuable information to have to be able to trace bad actors.

Reply:
Oh my bad, missed that it's just in the logs, not the response. lgtm